neo/rawio/bci2000rawio.py

```diff
@@ -47,7 +47,19 @@ def _parse_header(self):
             dtype = file_info['DataFormat']
             units = 'uV'
             gain = param_defs['SourceChGain']['value'][chan_ix]
+            
+            if isinstance(gain, str):
+                r = re.findall('(\d+)(\D+)', gain)
```

Comment:
I think you need to use a raw string here: `r = re.findall(r'(\d+)(\D+)', gain)`